### ISSUE TITLE
default user and roles created, every time application restarted

### ISSUE THREAD

**Issue by kukubadze** [aspnetboilerplate/module-zero-core-template#116]
```
public class HostRoleAndUserCreator
    {
....
        private void CreateHostRoleAndUsers()
        {
            var adminRoleForHost = _context.Roles.IgnoreQueryFilters().FirstOrDefault(r => r.TenantId == null && r.Name == StaticRoleNames.Host.Admin);
            if (adminRoleForHost == null)
            {
                adminRoleForHost = _context.Roles.Add(new Role(null, StaticRoleNames.Host.Admin, StaticRoleNames.Host.Admin) { IsStatic = true, IsDefault = true }).Entity;
                _context.SaveChanges();
            }
            //may be _context.Users.IgnoreQueryFilters() ?
            var adminUserForHost = _context.Users.FirstOrDefault(u => u.TenantId == null && u.UserName == AbpUserBase.AdminUserName);
            if (adminUserForHost == null)
            {
...
```

**Comment by acjh** [comment]
Did you try if your suggestion works?

**Comment by hikalkan** [comment]
Yes, we should add IgnoreQueryFilters to seed queries.

**Comment by hikalkan** [comment]
https://github.com/aspnetboilerplate/module-zero-core-template/pull/117

**Comment by acjh** [comment]
Fixed by #126
